web/utils/pb.ts

```diff
@@ -2,9 +2,9 @@ import PocketBase, { type RecordModel } from 'pocketbase'
 import { useRuntimeConfig } from '#app'
 import type { Item } from '#imports'
 
-const config = useRuntimeConfig()
-
-export const pb = new PocketBase(config.public.apiBase)
+export function usePocketBase() {
+  return new PocketBase(useRuntimeConfig().public.apiBase)
```

Comment:
I am afraid this is not going to improve the performance, and we are still going to need global object to subscribe for realtime events, so I just think something like this would work:

```
let _pb: PocketBase | null = null

export function usePocketBase(): PocketBase {
  if (!_pb) {
    _pb = new PocketBase(useRuntimeConfig().public.apiBase)
  }
  return _pb
}
```